pubsub/testing.go

```diff
@@ -59,7 +75,7 @@ func (s *FakeSubscription) Cancel() {
 // Manipulators
 
 // Post posts a new message to this subscription.
-func (s *FakeSubscription) Post(msg *pubsub.Message) {
+func (s *FakeSubscription) Post(msg *FakeMessage) {
```

Comment:
shouldnt this take a message, not a fakemessage?

Reply:
Thanks, sorry, that was a remnant from before I wrapped Message.